packages/table/src/regions.ts

```diff
@@ -119,6 +119,17 @@ export class Regions {
      *     }
      *
      * In this case, this method would return RegionCardinality.FULL_COLUMNS.
+     *
+     * If both rows and columns are unbounded, then the region covers the
+     * entire table. Therefore, a region like this:
+     *
+     *     {
+     *         rows: null,
+     *         cols: null
```

Comment:
@themadcreator does selection explicitly speak `null`? or does it do `undefined` too?